hips/tiles/surveys.py

```diff
@@ -127,7 +132,12 @@ def tile_format(self) -> str:
     @property
     def base_url(self) -> str:
```

Comment:
Try to load properties from http://alasky.u-strasbg.fr/DSS/DSS2-NIR/properties for instance